aptos-move/aptos-transaction-simulation/src/account.rs

```diff
@@ -0,0 +1,774 @@
+// Copyright © Aptos Foundation
```

Comment:
Note: this file is not new. It was moved over from `e2e-tests` to break a dependency cycle.